packages/mobile/src/components/button/styles.ts

```diff
@@ -1,10 +1,60 @@
 import { StyleSheet } from 'react-native';
+import { ButtonType, ButtonView } from '~/common/enums/enums';
+
+//mock before update global theme color
+enum AppColor {
```

Comment:
There is a `ColorPalette` enum in shared package

Reply:
updated